website/demos/ContextMenu.tsx

```diff
@@ -20,13 +20,16 @@ css`
       border-radius: 0.25rem;
       color: #373a3c;
       font-size: 16px;
-      margin: 2px 0 0;
-      min-width: 160px;
+      margin-block-start: 2px;
+      margin-inline-end: 0;
+      margin-block-end: 0;
```

Comment:
`margin: 2px 0 0;` is a shortcut for all 4 sides, looks like we're missing one side here
![image](https://user-images.githubusercontent.com/567105/151982776-da5b4a71-2630-4134-84cf-51cbe8a9f26b.png)